openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/startup/startup-controller.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, Injector } from '@angular/core';
+import { startupSequence } from './startup-sequence';
+
+export type StartupControllerState = true | 'in-progress' | false | 'error';
```

Comment:
Would this work the same?
```suggestion
export type StartupControllerState = boolean | 'in-progress' | 'error';
```

Reply:
Yeah it should; not sure why I did it this way.